specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/sqlX.json

```diff
@@ -0,0 +1,261 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved sql accelerator service resource.",
+            "schema": {
+              "$ref": "#/definitions/ServiceGetResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBSqlAcceleratorGet": {
+            "$ref": "./examples/CosmosDBSqlAcceleratorGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "SqlAcceleratorService"
+        ],
+        "description": "Creates the sql accelerator service for a Cosmos DB account.",
+        "operationId": "SqlAccelerators_CreateOrUpdate",
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "name": "createUpdateServiceResourceParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ServiceResourceCreateUpdateParameters"
+            },
+            "description": "The sql accelerator service to create for the current database account."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Creation of sql accelerator service will complete asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/ServiceGetResults"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "cosmos-db.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CosmosDBServiceResourceCreate": {
+            "$ref": "./examples/CosmosDBSqlAcceleratorCreate.json"
```

Comment:
Can we change all the SqlAccelerator to Sqlx?

Reply:
Sure, I would do that.